app/notes/constants.js

```diff
@@ -0,0 +1,7 @@
+(function() {
+    'use strict';
+
+    angular
+        .module('app.core')
+        .constant('http://localhost:3030/', LocalHost);
+})();

```

Comment:
You're getting an app.core module that hasn't been created. This should also probably be in its own folder since it's a separate module. Probably app/core/core.module.js